### DISCUSSION TITLE
Wrong format on finshed paged in github for Android

### DISCUSSION THREAD

**Discussion by VistaQvQ** [community/community#39556] [Mobile]
### Select Topic Area

Bug

### Body

![Screenshot_2022-11-17-09-31-54-58_40deb401b9ffe8e](https://user-images.githubusercontent.com/23132799/202888145-0703d6ac-a660-4246-884f-c86e0d23e9c9.jpg)
Above the picture ,here are Chinese words showed  in the page after I download files form github for Android: "%d个文件已下载完毕"(%d files had beed downloaded ). Whitch was showed in a wrong format. "%d" should be replaced by the number of file(s) I had already download to my device, but it didn't work.

Version: V1.71.1-beta(10074)

**Comment by taki-on** [comment]
Thank you for sharing the issue! I will forward this issue to the GitHub Android team 🙇

**Comment by taki-on** [comment]
@VistaQvQ We think the screen is not from the GitHub Android app, but it is probably from the Android OS. If the screen appears for any download actions on Android, can you try it with another app and check if you see the format string is still there?

**Reply by VistaQvQ** [reply]
I connect the PC and try ADB tools, and it said the screen comes from chrome for Android. and end the issue occurred when the device's default language was set to Chinese, everything ok if English 😂
